tests/e2e/README.md

```diff
@@ -0,0 +1,84 @@
+# Sail Operator end-to-end test
+
+This end-to-end test suite utilizes Ginkgo, a testing framework known for its expressive specs (reference: https://onsi.github.io/ginkgo/). The setup for the test run is similar to the upstream Istio integration tests:
+* In the case of kind execution, it relies on the upstream script `kind_provisioner.sh` and `integ-suite-kind.sh`, which are copied from the `github.com/maistra/istio` repository to set up the kind cluster used for the test.
```

Comment:
Not sure if you need to mention this, but if you do, the correct source is istio/common-files, not maistra/istio.